book/src/getting-started/3-exposing-to-qml.md

```diff
@@ -26,16 +26,22 @@ There are two notable changes compared to a normal Qt application though:
 {{#include ../../../examples/qml_minimal/cpp/main.cpp:book_qml_register}}
 ```
 
-For every QObject subclass that we define in Rust, CXX-Qt will generate a corresponding C++ class.
-This class is included by the first code snippet.
-They will always be in the `cxx-qt-gen/include/` include path and use the snake_case naming convention.
+For every `#[cxx_qt::bridge]` that we define in Rust, CXX-Qt will generate a corresponding C++ header file.
+This file is included by the first code snippet.
+They will always be in the `cxx-qt-gen/` include path and use the snake_case naming convention.
+The name of the header file will be the name of the Rust module of your `#[cxx_qt::bridge]`, followed by `.cxxqt.h`.
```

Comment:
we also later (in the qobject/macro.md file need to explain how you can override this with the `cxx_file_stem` attribute)

Reply:
Yup, but I didn't want to mention it in the Getting-Started guide yet.
Best to avoid complexity where possible.

Reply:
sure, was just a note for the `macro.md` book page later :-)